service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -480,10 +505,10 @@ private FurtherDirections generateOrderFurtherDirections() {
                 .build();
         }
 
-        private DocumentReference expectedDocument() {
+        private DocumentReference expectedDraftDocument() {
             return DocumentReference.builder()
                 .binaryUrl(document.links.binary.href)
-                .filename(document.originalDocumentName)
+                .filename("draft-file.pdf")
```

Comment:
any reason for the change?

Reply:
good spot this actually doesn't need to change

Reply:
changed back